README.md

```diff
@@ -510,6 +511,35 @@ The user can either `sign up` or `sign in`
 final authResponse = await Descope.password.signUp(loginId: 'desmond_c@mail.com', password: 'cleartext-password',
     details: SignUpDetails(name: 'Desmond Copeland'));
 ```
+## Error Handling via `DescopeException`
+
+All of the authentication operations will throw a `DescopeException` in the event an error
+is encountered. There are several ways to catch and handle a `DescopeException` thrown by a Descope SDK
+operation, and you can use whichever one is more appropriate in each specific use case.
```

Comment:
```suggestion
All authentication operations throw a `DescopeException` in case of a failure. There are several
ways to catch and handle a `DescopeException` thrown by a Descope SDK operation, and you
can use whichever one is more appropriate in each specific use case.
```